### PR TITLE
Add the movfuscator

### PR DESCRIPTION
**Pull request by jeremy-rifkin**

### CODE REVIEW

**Comment by jeremy-rifkin** [comment]
Manually ran a bespoke build and installed locally. This should be good to go.

**Comment by jeremy-rifkin** [comment]
Ok - now the patches required to get the movfuscator working at a basic level have been sorted out and this is actually good to go

**Review by partouf — APPROVED** [review]

File: bin/yaml/c.yaml [hunk 1]
```diff
@@ -82,6 +82,14 @@ compilers:
         check_file: chibicc
         targets:
           - main
+      movfuscator:
+        type: nightly
+        s3_name: movfuscator-{{name}}
```
Comment (partouf) [inline bin/yaml/c.yaml:87]: isn't this the default?

Comment (jeremy-rifkin) [reply bin/yaml/c.yaml]: Looks like it, thanks!

**Pull request merged**